.circleci/config.yml

```
@@ -217,13 +217,17 @@ jobs:
       - reconfigure_origin_remote
 
       - run:
-          name: Check for updates of IntelliJ EAP
-          command: ./gradlew updateEapBuildNumber
+          name: Check for updates of IntelliJ versions
+          command: ./gradlew updateIntellijVersions
       - create_pull_request_if_files_changed:
-          branch_name_prefix: update-eap
-          branch_name_command: &extract_eap grep -Po '(?<=eapOfLatestSupportedMajor=).*(?=-EAP-SNAPSHOT)' intellijVersions.properties
-          commit_message_prefix: Update EAP build number to
-          commit_message_command: *extract_eap
+          branch_name_prefix: update-intellij-versions
+          branch_name_command: md5sum intellijVersions.properties | cut -d' ' -f 1
+          commit_message_prefix: Update IntelliJ versions ->
+          commit_message_command: git diff | grep -P '\+' |
```

Hmmm I suppose you'd like to see the lines **starting** with a plus, right?
```suggestion
          commit_message_command: git diff | grep '^+' |
```
Although it won't make much difference anyway, there shouldn't be any pluses other than the initial one anyway

Also, `-P` aka `--perl-regexp` is redundant, you can just use regular syntax, where `+` is a character like any other (unlike with _extended_ syntax, `grep -E`, where `+` stands for "1 or more occurrence")